src/app/pages/component-viewer/component-api.html

```diff
@@ -4,22 +4,22 @@
   </span>
 
   <!--
-  The span container is necessary for the layout to display properly. When
+  This container is necessary for the layout to display properly. When
   a component has multiple API docs, they need to be joined together in the
   same container so that they display one after another.
   -->
-  <span>
+  <div class="docs-component-api">
```

Comment:
This causes the container to start with 80% width instead of something like 50px width that wraps the message.

I changed it from a `span` since the layout needs to be block and not inline.